data/human/human missions.txt

```diff
@@ -2514,6 +2514,139 @@ event "terraforming follow-up"
+	source "Desi Seledrak"
+	destination "Var' Kayi"
+	passengers 2
+	blocked "You need <capacity> in order to take on this mission."
+	to offer
+		has "Terraforming 13: done"
+	on offer
+		conversation
+			`It is not long before Iriket arrives. You introduce Amy to her, and explain why you have brought her here. "I would be [glad, honored] to answer any questions about our work," she agrees.`
+			`	What follows is an extensive discussion about how the Wanderers go about planning the terraforming process for a world. Topics range from the earliest considerations as to what the goal for each planet should be to the role of their artificial minds in helping to monitor and guide the process in the long term. Your translator has some difficulty with the more technical aspects of the discussion, which leads Iriket to rephrase some things many times. Throughout it all, Amy is taking copious notes. Iriket also explains some of what they have been doing to improve the climate on this world.`
+			`	"The gases [released, produced] by the factories will help the atmosphere retain more heat, which will in turn begin to melt the glaciers and return water to the [ecosystem, environment]. Over time, this will [mix, interact] with the abundant ash, helping to [restore, rejuvenate] the soils all over this world. In time, it will become [fruitful, bountiful] again."`
+			`	"Won't that take hundreds or thousands of years?" Amy asks.`
+			`	"Indeed," says Iriket. "But, to attempt to [accelerate, advance] the process by more [direct?] means would [harm, deteriorate] the product. While some worlds may [benefit from, require] a more [extreme, radical] change to begin their restoration, we do not believe it is [necessary, appropriate] here. It is [important, our goal] that a world at the conclusion of our work not only be [pristine, healthy] in that moment, but that it be [sustainable?] without additional [intervention, support]."`
+			`	Eventually, Amy asks "Are there any worlds where you have completed your work that I could see?" Iriket suggests <destination>, and asks you to take them there.`
+
+
+
+mission "Terraforming 15"
+	landing
+	name "Field Trip"
+	description "Take Tema'a Iriket and Amy to <destination> to retrieve records of Wanderer terraforming for Amy to study."
+	source "Var' Kayi"
+	destination "Vara K'chrai"
+	passengers 2
+	to offer
+		has "Terraforming 14: done"
+	on offer
+		conversation
+			`You fly across large swathes of the planet's surface, giving Amy a view of all the greenery and the oceans here. Meanwhile, Tema'a Iriket describes the condition of this world when the Wanderers first began working on it. "The atmosphere was [close to, nearing] a [runaway, uncontrolled] greenhouse effect. Many species had recently gone extinct, or were very close to it. The oceans were also quite toxic. Now, the atmosphere is [stable, sustainable], and many of the species that were [on the brink?] of extinction now flourish."`
```

Comment:
I'm finding trouble verifying this, but would the American spelling be swaths (without the e)?

Reply:
```suggestion
			`You fly across large swaths of the planet's surface, giving Amy a view of all the greenery and the oceans here. Meanwhile, Tema'a Iriket describes the condition of this world when the Wanderers first began working on it. "The atmosphere was [close to, nearing] a [runaway, uncontrolled] greenhouse effect. Many species had recently gone extinct, or were very close to it. The oceans were also quite toxic. Now, the atmosphere is [stable, sustainable], and many of the species that were [on the brink?] of extinction now flourish."`
```